src/components/views/EResource.js

```diff
@@ -68,11 +85,14 @@ export default class EResource extends React.Component {
     if (isLoading) return <LoadingPane data-loading id="pane-view-eresource" {...paneProps} />;
 
     let EResourceViewComponent = Package;
+    let helperAppEndPoint;
 
     if (data.eresource?.class === resourceClasses.TITLEINSTANCE) {
+      helperAppEndPoint = 'titles';
```

Comment:
I think this should be in the route handler component. You're bleeding networking decision-making into the view components which should be avoided. Maybe just pass the `eresource` back instead of the `helperAppEndPoint`.